### DISCUSSION TITLE
Downloading ISO with a Torrent

### DISCUSSION THREAD

**Discussion by stephenhoos** [Security-Onion-Solutions/securityonion#4048] [Ideas]
Is there any way you could create a torrent of this file to download?  I have spotty internet and I find using bittorrent works very well for large files like this.

**Comment by weslambert** [comment]
Unfortunately, we don't intend on creating a torrent of/for this at this time.  You might consider installing using Centos 7 or Ubuntu 18.04 ISO and performing a network install as an alternative (although, not sure how much better that would end up being if internet is spotty 😃 ).
